recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+        - zlib
+      run_exports:
+        - {{ pin_subpackage('heavydb',  max_pin='x.x.x') }}
+    requirements:
+      build:
+        # c compiler is specified here to get run constraint pins correct, presumably..
+        - {{ compiler('c') }}
+        - {{ compiler('cgo') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+        - clangdev {{ llvm_version }}
+        # clang++ is used for generating the bytecodes of extension functions
+        - clangxx {{ llvm_version }}
+        - cmake
+        - llvmdev {{ llvm_version }}
+        - make
+        - ninja
+        - maven
+      host:
+        - arrow-cpp ={{ arrow_version }}=*{{ build_ext }}
+        - bisonpp  # [unix]
+        - blosc
+        - boost-cpp
+        - clangdev {{ llvm_version }}
+        - double-conversion
+        - flex  # [unix]
+        - fmt
+        - gflags
+        - glog
+        - llvmdev {{ llvm_version }}
+        - llvm {{ llvm_version }}
+        - libarchive
+        - libclang-cpp {{ llvm_version }}
+        - libevent  # [win]
+        - libgdal
+        - libkml
+        - libpng
+        - librdkafka
+        - libzlib
+        - ncurses   # [unix]
+        - openldap  # [unix]
+        # Workaround https://github.com/mamba-org/boa/issues/119 :
+        - openssl 1.1.1*
+        - snappy
+        - tbb
+        - tbb-devel
+        - thrift-cpp {{ thrift_version }}
+        - winflexbison  # [win]
+        - xerces-c
+      run:
+        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
+        - boost-cpp
+        - bzip2
+        # omnscidb Load-time UDF support calls clang++
+        - gxx_{{ target_platform }}  # [not win]
+        - libclang-cpp {{ llvm_version }}
+        - ncurses  # [not win]
+        - openjdk 8.*
+        - xz
+        - zlib
+        - {{ pin_subpackage('heavydb-common',  max_pin='x.x.x') }}
+      run_constrained:
+        - arrow-cpp-proc {{ arrow_proc_version }} {{ build_ext }}
+        - cudatoolkit >=11.0  # [cuda_compiler_version not in (undefined, "None")]
+
+    test:
+      commands:
+        # binary
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavydb         # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/initheavy       # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/heavysql        # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/KafkaImporter   # [unix]
+        - test -f ${PREFIX}/{{ install_base }}/bin/StreamImporter  # [unix]
+        - if not exist %PREFIX%\bin\heavydb.exe exit 1             # [win]
+        - if not exist %PREFIX%\bin\initheavy.exe exit 1           # [win]
+        - if not exist %PREFIX%\bin\heavysql.exe exit 1            # [win]
+
+        # Verify the activation scripts are in-place.
+        {% for state in ["activate", "deactivate"] %}
+        - test -f "${PREFIX}/etc/conda/{{ state }}.d/{{ PKG_NAME }}_{{ state }}.sh"  # [unix]
+        {% endfor %}
```

Comment:
Might be simpler to just write both activate and deactivate as separate lines, instead of a 2-item for loop.